third_party/rss_lib.BUILD

```diff
@@ -0,0 +1,70 @@
+# ----------------- BEGIN LICENSE BLOCK ---------------------------------
+#
+# INTEL CONFIDENTIAL
```

Comment:
Can you help to update the license with Apache license?

Reply:
@ycool thanks for your comments, this issue is resolved in the patch "remove-rss-lib-apollo-build-script-legal-disclaimer"